script/circleci

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+case $CIRCLE_NODE_INDEX in
+  0 )
+    bundle exec rake test
```

Comment:
Yeah but as @rafaelfranca pointed out, at least they have decent build matrixes. Is there a way to get around the queues using :dollar: ?